modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/processors/PathsProcessor.java

```diff
@@ -256,15 +260,31 @@ else if(model instanceof ComposedSchema) {
                 for (Schema innerModel : composedSchema.getAllOf()) {
                     updateRefs(innerModel, pathRef);
                 }
-            }if (composedSchema.getAnyOf() != null) {
-                for(Schema innerModel : composedSchema.getAnyOf()) {
+            }
+            if (composedSchema.getAnyOf() != null || composedSchema.getOneOf() != null) {
```

Comment:
AS mentioned above:

IIANM the changes here consider `oneOf` and `anyOf` as mutually exclusive, while they can coexist even if not so common. I would suggest to consider them both in the processing

Reply:
Made the processing to be mutually inclusive. 
Intuitively though, the definitions feel that they are mutually exclusive.